src/symbols/analyzer.rs

```diff
@@ -1349,8 +1379,8 @@ impl<'a> SymbolFiller<'a> {
             );
             self.fill_fn_decl(symbol, n);
             let symbol_id = symbol.symbol_id();
-            add_export(n.ident.sym.to_string(), symbol_id);
-            add_child(symbol_id);
+            module_symbol.add_export(n.ident.sym.to_string(), symbol_id);
+            module_symbol.add_child_id(symbol_id);
```

Comment:
In a future PR I'll add the concept of "symbol parents" so that we can go from something like a class method declaration symbol all the way up to the module symbol.

Reply:
This is in https://github.com/denoland/deno_graph/pull/324 (which also has a couple bug fixes for this PR)